### ISSUE TITLE
Big Frustrations with Mooltipass

### ISSUE THREAD

**Issue by immyburton** [limpkin/mooltipass#159]
Ok, so my mooltipass came today, after all the delays with shipping and email overload I was already pretty frustrated but was prepared to wipe the slate clean and give it a go.

But the frustrations of this project are overwhelming:
* You've sent 45 update emails and another 10 order emails. One even included a link to a python script you had used to try and fix something. **I mean seriously WFT?**
* The pin code entry system is a massive pain to use, entering a password is hard, so it's slow, so anyone could read it over my shoulder.
* The app doesn't work with Linux out of the box and itself offers no indication of how to fix it. **How hard would this be?**
* The faq section on fixing this issue on the website is infuriating:
  * it closed every time I tried to click on the text to select it
  * The code isn't formatted as code, so it's not clear
  * Do you really think most Linux users know off the top of their head how to create a udev rule?
  * a link to a 10 line gist with instructions on how to fix this would have been very useful. **How hard would this be?**
* You seem to have lots of different projects inside the same git repo on github, multiple repos are free, much safer to use and much clearer for anyone browsing for information. Given that this is supposed to be a sustainable project not linked to one person's whims it seems very concerning that the one massive mess project is under a user's account not an organisation.**How hard would this be?**
* I sent 15 minutes try to use the mooltipass (and by the way it's a stupid name) to login to facebook (you've heard of it I assume? you sure as fuck haven't got proper unit tests interfacing with it). Having entered my password into numerous insecure places and go nowhere I abandoned the effort.
* I then create a new user on github with a new password I didn't care about and tried to use the mooltipass to login. 
  * It took me 15 minutes of messing around to get it to actually login.
  * Even once I got it working the thing kept asking to enter my credentials into the github anonymous landing page and the github authenticated landing page.
  * The "generate random password" feature doesn't seem to work at all
* So the UX of your thing with both the second most used website in the world and with the management system you use is beyond awful. **I mean seriously WFT?**

In short if you apply the same level of intelligence to safe guarding my passwords as you have to the rest of this project I must assume my facebook password will already be on safe on 50 different sites.

**Comment by limpkin** [comment]
immyburton,

I'll start this message by saying that github isn't a place to vent, the crowdfunding page might be.
We're terribly sorry you feel this way and you are welcome to return your unit to get a full refund.
To answer your points:
- pin code system: we've never had any bad feedback about it from all of our beta testers that have been using it for less than a year now
- linux: are you referring to the udev rule? it has been included in the FAQ, and we can add it to our app (that is still work in progress) without problem. We thank you for that suggestion, even if it is phrased this way
- repository organization: people have very different ways of structuring their projects, we have heard points for and against our way
- have you read our user manual that details how to enter logins & password on the possible websites that our extension might have problem with?

All our features have been thoroughly tested by our beta testers. We do feel bad that you didn't take some time to meet us on #mooltipass to talk about the issues you were having so we could figure out a way to make the process easier for future people getting to know the mooltipass.
In all our updates we emphasized the fact that we were available for a chat. We actually wanted to be here to prevent the frustrations you could have.
Once again, you're welcome to return your unit for a full refund.

**Comment by NoahAndrews** [comment]
Are you sure that you're staying **within** the blue wheel boundaries? I'm able to log in quickly, now that I know about that limitation. I'm actually a big fan of the PIN entry system. It allows for hex values, I find it to be quick, it works with the input system, and most importantly, fingerprints mean nothing to someone gaining access to the device.

Facebook login works exactly as expected for me.
